cmd/nightfalldlp/main.go

```diff
@@ -28,7 +29,9 @@ func main() {
 
 func run() error {
 	ctx := context.Background()
-	diffReviewClient, err := CreateDiffReviewerClient(&http.Client{})
+	flageValues := flag.Parse()
+
```

Comment:
So it looks like we instantiate `flagValues` by calling `flag.Parse` which sets the debug flag to `false`, but how can we set debug to `true`, or will we never need to?

Reply:
In the flag client we indicate that there is a flag called `debug` or `-d` [here](https://github.com/nightfallai/jenkins_test/pull/1/files#diff-5e93389ce0bcba7cc920f68b25888b55R14). And then when we parse the flags [here](https://github.com/nightfallai/jenkins_test/pull/1/files#diff-5e93389ce0bcba7cc920f68b25888b55R16) the `pflag` lib will set the `Debug` value on the `Values` struct to the appropriate value.